kouncil-frontend/libs/common-login/src/lib/change-password/common-change-password.component.ts

```diff
@@ -0,0 +1,77 @@
+import {ChangeDetectionStrategy, Component, EventEmitter, Input, Output} from '@angular/core';
+import {FormBuilder, FormControl, FormGroup, Validators} from "@angular/forms";
+import {Backend} from "@app/common-model";
+
+@Component({
+  selector: 'app-common-change-password',
+  template: `
+    <div class="icon-login-container"
+         [ngClass]="backend === 'SERVER' ? 'icon-login-container-desktop' : 'icon-login-container-demo'">
```

Comment:
Same like i common-login. Use composition.
It will make component less smart and boundary context will be better defined.